apps/fluent-tester/src/TestComponents/LinkExperimental/InlineLinksTest.win32.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { TextV1 as Text } from '@fluentui/react-native';
+
+// Platform.select not available for win32
+// Forking test instead
+export const InlineLinks: React.FunctionComponent = () => {
+  return <Text>Inline links are not supported on win32.</Text>;
+};
```

Comment:
Nit: I've given feedback in other PRs to try and fork tests as little as possible for consistency of people reading code (and also try to standardize where we fork by platform). To that effect, I wonder if it's better to just put this check in the default file instead of forking for win32. Not a strong preference either way since we haven't documented how we want to handle platform specific tests yet.

Reply:
I'm in camp fork tests 😅Personally I think the code is eventually going to get really messy with platform checks. I'll WF for now, we can decide when documentation is written up.